base/applications/mscutils/servman/export.c

```diff
@@ -73,9 +79,12 @@ SaveServicesToFile(PMAIN_WND_INFO Info,
                               sizeof(WCHAR) * dwTextLength,
                               &dwWritten,
                               NULL);
+                }
 
+                if (k != 4)
+                {
```

Comment:
Ideally it would be using `k < _countof(Columns) - 1`, but this would require some more changes. Suggesting this:
```suggestion
                if (k < 4)
                {
                    /* Do not add separator after the last table cell */
```

Reply:
Ok I have pushed a fix for you. 🙂